src/start_arguments.cpp

```diff
@@ -0,0 +1,46 @@
+#include "start_arguments.h"
+
+//Initialize extern variables
+bool startargs::doip_flag = false;
+bool startargs::can_flag = false;
+std::string startargs::can_device;
+
+/**
+ * Parses the arguments when starting the car simulator.
+ * Usage:   ./car-simulator -d vcan0    for doip and can simulation
+ *          ./car-simulator             for doip and can simulation
+ *          ./car-simualtor -d          for doip simulation
```

Comment:
A small spelling mistake occurs at the above line for the arguments to start the car simulator. Should be "./car-simulator -d".

Reply:
thank you, we will change that